src/components/filter/Filter.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import './Filter.css';
+const Filter = () => {
+  return (
+    <aside className="filter-section">
+      <form className="filter-container" action="">
+        <div className="filter-head-section d-flex">
+          <h3 className="filter-title">Filters</h3>
+          <button className="btn btn-link-primary filter-clear-btn">
+            Clear
+          </button>
+        </div>
+        <div className="filter-elements filter-price-element">
+          <h3>Price</h3>
+          <div className="filter-input-container">
+            <input
+              type="range"
+              min="$50"
+              max="$35000"
+              className="range-selector filter-range-selector"
+            />
+            <div className="filter-price-bottom-container">
+              <span>$50</span>
+              <span>$35000</span>
+            </div>
+          </div>
+        </div>
+        <div className="filter-elements filter-category-element">
+          <h3>Category</h3>
+          <div className="filter-input-container">
+            <label className="margin-b-5">
+              <input
+                name="digital_art_checkbox"
+                id="digital_art_checkbox"
+                type="checkbox"
+                value="Digital art"
+                className="checkbox"
+              />
+              Digital Art
+            </label>
+
+            <label className="margin-b-5">
+              <input
+                name="photography_checkbox"
+                id="photography_checkbox"
+                type="checkbox"
+                value="Photography"
+                className="checkbox"
+              />
+              Photography
+            </label>
+            <label className="margin-b-5">
+              <input
+                name="music_checkbox"
+                id="music_checkbox"
+                type="checkbox"
+                value="music"
+                className="checkbox"
+              />
+              Music
+            </label>
+            <label className="margin-b-5">
```

Comment:
you can create one componenet for checkbox input and use that, as you are repeating same code and just changing values.